_posts/2018-12-28-cardioid.md

```diff
@@ -8,6 +8,5 @@ Cardioid is a cardiac multiscale simulation suite spanning from subcellular mech
 This project's history goes back a few years -- it was a finalist for the 2012 Gordon Bell Prize -- but only now is the code available as open source. Initially developed by a team of LLNL and IBM scientists, Cardioid divides the heart into a large number of manageable subdomains. This replicates the electrophysiology of the human heart, accurately simulating the activation of each heart muscle cell and cell-to-cell electric coupling.
 
 - [Cardioid on GitHub](https://github.com/llnl/cardioid) - for best results, compile with [Spack](https://github.com/spack/spack)
-- Video: [The Cardioid Project: Simulating the Human Heart](https://computing.llnl.gov/cardioid-project-simulating-human-heart-0)
```

Comment:
Could we replace this with a YouTube link? I believe this is the same video. https://youtu.be/KQIqLb9kYtA

Reply:
Done!